### PR TITLE
Fix typos in assign-pod-node.md

### PR DESCRIPTION
**Pull request by anavalo**
This PR fixes small typos in the text for consistency.

### CODE REVIEW

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 1]
```diff
@@ -632,7 +632,7 @@ The following are all the logical operators that you can use in the `operator` f
 
 The following operators can only be used with `nodeAffinity`.
 
-|    Operator    |    Behaviour    |
+|    Operator    |    Behavior    |
```
Comment (niranjandarshann) [inline content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:635]: @anavalo   both the words are same noun Its depend on the Region to region like in In British English, you use “behaviour.” In American English, you use “behavior" .

So Here the word Behaviour is not required to be changes as we are using same in the entire docs.

**Comment by niranjandarshann** [comment]
@anavalo  Thank You for your Contribution. Small PR can bring a beautiful change in the docs. So keeps on contributing. Try to find all the typo error in the entire page and raise the PR. It will be good and effective for us to go ahead in one go.

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md (hunk 1 above)
Comment (anavalo) [reply content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:635]: In this `md`, both nouns are used. The correction is to use the same noun consistently on this page.

Comment (niranjandarshann) [reply content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:635]: Yes I seen the word Behavior is used consistently in this page. So its look good to me.

**Review by alculquicondor — COMMENTED** [review]
/lgtm

**Review by nate-double-u — COMMENTED** [review]
Hi [@anavalo](https://github.com/anavalo), thanks for this, and welcome!
Please be careful to [avoid trivial edits](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits) in the future. Here are things to consider when making minor changes:
* Can the file be improved further?
* Does the small change greatly enhance the quality of the content?

/approve

**Pull request merged**